templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
+[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
+
+ocpath='{{ owncloud_home }}'
+htuser='{{ owncloud_user }}'
+htgroup='{{ owncloud_group }}'
```

Comment:
Perhaps a small suggestion (which could eventually be added to Script Policy if it ever appears): Ansible environment and script environment are different things, and I like to pass variables from one to the other in the above fasion. Then, the rest of the script uses only Bash variables which hopefully should make it easier to review and understand by the user.

Ultimately, scripts could be just normal files (not templates), which source their configuration variables from external files, akin to how init scripts are doing with `/etc/default/<service>` files. These configuration files then could be templated by Ansible.